protocol-units/bridge/move-modules/sources/atomic_bridge_counterparty.move

```diff
@@ -146,154 +149,83 @@ module atomic_bridge::atomic_bridge_counterparty {
             },
         );
     }
-
     
-    #[test(creator = @atomic_bridge)]
-    fun test_init_module(
-        creator: &signer,
-    ) acquires BridgeTransferStore, BridgeConfig {
-        let owner = signer::address_of(creator);
-        let moveth_minter = @0x1; 
-        init_module(creator);
+    #[test_only]
+    public fun set_up_test(origin_account: signer, resource: &signer) {
```

Comment:
Because it's a test function, it's okay to pass in multiple signers. Just not with production functions.

Here's the example I modeled it after from Aptos' `minting.move`:

```
    #[test_only]
    public fun set_up_test(origin_account: signer, collection_token_minter: &signer, aptos_framework: signer, nft_receiver: &signer, timestamp: u64) {
        // set up global time for testing purpose
        timestamp::set_time_has_started_for_testing(&aptos_framework);
        timestamp::update_global_time_for_test_secs(timestamp);

        create_account_for_test(signer::address_of(&origin_account));

        // create a resource account from the origin account, mocking the module publishing process
        resource_account::create_resource_account(&origin_account, vector::empty<u8>(), vector::empty<u8>());

        init_module(collection_token_minter);

        create_account_for_test(signer::address_of(nft_receiver));
    }
```

Here instead of `resource` they name it `collection_token_minter` and it does need to be passed into `init_module`.

I renamed it `fa_minter` which probably isn't perfect either. Probably good for us to discuss naming conventions and clean up the names and removing unneeded names when we have more consensus on that.